### PR TITLE
[Doc] HTTP error codes 401, 403, 413 & 429

### PR DESCRIPTION
**Pull request by pascal-vs**
Add documentation for HTTP error codes 401, 403, 413 and 429.

### CODE REVIEW

File: documentation/slate/source/includes/_errors.md [hunk 1]
```diff
@@ -37,19 +56,28 @@ The are two possible 40x http codes:
...
-| unable_to_parse   | When you use a mal-formed custom filter                     |
-| unknown_api       | When you request an inexistant endpoint                     |
-| bad_format        | When you provide ill-formated parameter (coord for example) |
-| config_exception  | When you try to configure an external service (street network back-end or autocomplete) with wrong parameters |
+When the response generated by the request exceeds 10 MB (few APIs are not yet concerned by this limitation but this may change in the future).
 
+Tips to work around this error:
+
+* [paginate](#paging)
+* reduce the parameter `depth`
```
Comment (woshilapin) [inline documentation/slate/source/includes/_errors.md]: Maybe, not sure :shrug: 

```suggestion
* reduce the value of the parameter `depth`
```

File: documentation/slate/source/includes/_errors.md [hunk 2]
```diff
@@ -37,19 +56,28 @@ The are two possible 40x http codes:
...
+* on large `journeys`, reduce the time frames, durations, _etc_
```
Comment (woshilapin) [inline documentation/slate/source/includes/_errors.md]: `time_frame_duration` is one parameter. Another one of interest would `min_nb_journeys`.

```suggestion
* on large `journeys`, reduce the `time_frame_duration`, `min_nb_journeys`, _etc_
```

**Review by woshilapin — APPROVED** [review]

File: documentation/slate/source/includes/_errors.md (hunk 2 above)
Comment (pascal-vs) [reply documentation/slate/source/includes/_errors.md]: did you prefer more precision on the workaround? Do you agree the last change?

**Review by woshilapin — APPROVED** [review]

File: documentation/slate/source/includes/_errors.md (hunk 2 above)
Comment (woshilapin) [reply documentation/slate/source/includes/_errors.md]: Good enough for me. Thanks for the nice documentation.

**Pull request merged**